### PR TITLE
Track Info dialogs: move metatdata buttons below color picker

### PR DESCRIPTION
**Pull request by ronso0**

### CODE REVIEW

File: src/library/dlgtrackinfomulti.ui [hunk 1]
```diff
@@ -437,7 +429,7 @@
        </widget>
       </item>
 
-      <item row="0" column="0">
+      <item row="10" column="0">
```
Comment (Swiftb0y) [inline src/library/dlgtrackinfomulti.ui:432]: do you know why this was `row="0"` previously?

Comment (ronso0) [reply src/library/dlgtrackinfomulti.ui:432]: 🤷‍♂️ 
items with the same position are stacked, that's why we didn't notice.

Comment (Swiftb0y) [reply src/library/dlgtrackinfomulti.ui:432]: so the spacer was overlayed on another object somewhere near the top of the table, effectively not existing?

Comment (ronso0) [reply src/library/dlgtrackinfomulti.ui:432]: yup. that's what I noticed with QLabels when I messed up column/row values.

Comment (Swiftb0y) [reply src/library/dlgtrackinfomulti.ui:432]: Great. thanks for the clarification then.

**Review by Swiftb0y — APPROVED** [review]
LGTM. Thank you

**Comment by ronso0** [comment]
Thanks for the quick review!

**Pull request merged**